internal/testing/runchecks.sh

```diff
@@ -77,9 +77,14 @@ fi
 # ./internal/testing/alldeps
 #
 # Whenever project dependencies change, rerun ./internal/testing/listdeps.sh
-./internal/testing/listdeps.sh | diff ./internal/testing/alldeps - || {
-  echo "FAIL: dependencies changed; compare listdeps.sh output with alldeps" && result=1
-}
+#
+# Run this only on the latest version of Go, since module behavior may differ
+# across versions.
+if [[ $(go version) == *1\.12* ]]; then
```

Comment:
Nit: there's `TRAVIS_GO_VERSION` which may be a bit cleaner

Reply:
I saw that, but it's undocumented what the content of that env var is. I'd rather have an expression I can test locally.